### PR TITLE
area(Getting-Started): Updated Getting Started Section

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

**What this PR does / why we need it**: Updated Getting Started Section to adjust to new structure

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [X] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/getting-started/installation.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+sidebar_label: Installation
+---
+
+---
+
+## Prerequisites
+
+Before deploying LitmusChaos, make sure the following items are there
+
+- Kubernetes 1.15 or later
```
Comment (imrajdas) [inline website/docs/getting-started/installation.md]: We are supporting for 1.17 or later

cc: @ksatchit

File: website/docs/getting-started/installation.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+```
+
+#### Step-2: Create the namespace on which you want to install Litmus ChaosCenter
+
+- The litmus infra components will be placed in this namespace.
+
+> The ChaosCenter can be placed in any namespace
+
+```bash
+kubectl create ns <LITMUS_PORTAL_NAMESPACE>
```
Comment (imrajdas) [inline website/docs/getting-started/installation.md]: If Replace <LITMUS_PORTAL_NAMESPACE> with litmus, the user experience will  be good.

**Review by imrajdas — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/installation.md (hunk 1 above)
Comment (S-ayanide) [reply website/docs/getting-started/installation.md]: Have updated the same in prerequisites

File: website/docs/getting-started/installation.md (hunk 2 above)
Comment (S-ayanide) [reply website/docs/getting-started/installation.md]: Sure, have replaced this in Getting Started Installation for UX.

The approach for Advance Installation have been kept using <LITMUS_PORTAL_NAMESPACE> with this note
> If installing Litmus in any other namespace apart from litmus namespace, make sure to change the same in the manifest https://litmuschaos.github.io/litmus/2.0.0-Beta/litmus-2.0.0-Beta.yaml.

**Review by imrajdas — APPROVED** [review]
LGTM

**Pull request merged**